### PR TITLE
Bump version to 0.98.0

### PR DESCRIPTION
**Pull request by bwoebi**
The one point zero will come soon :-)

### CODE REVIEW

File: package.xml [hunk 1]
```diff
@@ -62,45 +62,73 @@
...
+### Added
+- Roadrunner appsec support #2443
+- Provide server.response.body #2489
+- Support json/xml request bodies #2499
+
+### Changed
+- Change ip header parsing/header reporting #2503
+
+### Fixed
+- Fix blocking on PHP 7.0-7.1 ZTS #2502
```
Comment (Anilm3) [inline package.xml:128]:
```suggestion
- Fix blocking on PHP 7.0-7.1 ZTS #2502
- Ensure logs are committed to disk upon shutdown #2495 
```

File: package.xml [hunk 2]
```diff
@@ -62,45 +62,74 @@
...
+
+Windows support is in beta. We recommend carefully deploying the tracer to staging environments and testing it there before putting it to production. We are looking towards stabilizing Windows quickly, so any feedback and issue reports are welcome.
+
 ### Added
-- Add span links capabilities to the OTel API #2451
-- Add exec integration #2361, #2455, #2459
+- Enhance logging #2490
+  - This features two new logging options:
+    - `datadog.trace.log_level` allows more fine-grained control over logging (`error`, `warn`, `info`, `debug`, `trace`). It follows the RUST_LOG standard. Note that `datadog.trace.debug` still takes precedence if set.
+    - `datadog.trace.log_file` allows specifying a dedicated target file for any datadog trace logs instead of the default php error location.
```
Comment (PROFeNoM) [inline package.xml:78]: Question: Have we updated the documentation?

File: package.xml [hunk 3]
```diff
@@ -62,45 +62,73 @@
...
+- Update dependencies and pin `ahash` #2479
+- Fix clippy::single-match #2482
+- Update to libdatadog 6 #2513
+- Refactor: use SystemSettings in locals #2487
+- Fix accidentally skipped tests #2517
 
 ## Application Security Management
-### Internal changes
-- Change schema extraction zip compression to PHP_ZLIB_ENCODING_GZIP #2447
+### Added
```
Comment (Anilm3) [inline package.xml:119]:
```suggestion
This release introduces support for the Roadrunner application server and extends threat detection and blocking on XML and JSON request payloads. Request and response payloads are now also used to expand the already existing API security features.

### Added
```

**Review by Anilm3 — APPROVED** [review]

**Review by realFlowControl — APPROVED** [review]

File: package.xml (hunk 2 above)
Comment (bwoebi) [reply package.xml:78]: :-)

Yeah, I guess I should PR that now.

**Pull request merged**